pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -329,16 +339,16 @@ func (p *Provider) InstallTinkerbellStack(ctx context.Context, cluster *types.Cl
 			Manifest:    bundle.Tink.Manifest.URI,
 			Deployments: []string{"tink-server", "tink-controller-manager"},
 		},
-		// {
-		// 	Name:        "boots",
-		// 	Manifest:    bundle.Boots.Manifest.URI,
-		// 	Deployments: []string{"boots"},
-		// },
-		// {
-		// 	Name:        "hegel",
-		// 	Manifest:    bundle.Hegel.Manifest.URI,
-		// 	Deployments: []string{"hegel"},
-		// },
+		{
```

Comment:
I can't find something that actually uses this slice entry. The only place that seems to reference it is an explicit exclusion on l. 361.

Reply:
The boots entry? It gets used when deploying boots on workload cluster